src/screens/EthToSub.tsx

```diff
@@ -200,6 +206,35 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 	);
 };
 
+function parseAmount(val: string|null) {
+	if (!val) {
+		return val;
+	}
+
+	return val.replace(/k/gi, '000').replace(/m/gi, '000000');
+}
+
+function parseReceiver(recv: string|null): string|null {
```

Comment:
recv should never be null because we initialise it as 'joshy'; It might end up being an empty string, which is falsy, so no parsing will be done.

```suggestion
function parseReceiver(recv: string): string {
```